packagingbuild/centos8/Dockerfile

```diff
@@ -0,0 +1,101 @@
+FROM centos:8
+
+# Download tools
+RUN yum -y install \
+    ca-certificates \
+    curl \
+    wget
+
+RUN yum -y install \git \
+    mercurial \
+    openssh \
+    subversion
+
+    
+RUN yum install -y yum-utils \
+    && dnf config-manager --enable PowerTools \
+    && yum install -y epel-release \
+    && yum -y install \
+    ImageMagick \
+    ImageMagick-devel \
+    libyaml-devel \
+    && dnf config-manager --disable PowerTools \
+    && yum remove -y epel-release yum-utils
+    
+
+# Build tools
+RUN yum -y install \
+    autoconf \
+    automake \
+    bzip2 \
+    bzip2-devel \
+    file \
+    gcc \
+    gcc-c++ \
+    glib2-devel \
+    glibc-devel \libcurl-devel \
+    libevent-devel \
+    libffi-devel \
+    libjpeg-devel \
+    libtool \
+    libwebp-devel \
+    libxml2-devel \
+    libxslt-devel \make \
+    mysql-devel \
+    ncurses-devel \
+    openssl-devel \
+    patch \
+    postgresql-devel \
+    readline-devel \
+    sqlite-devel \
+    xz \
+    xz-devel \
+    zlib-devel
+
+# St2 package build debs
+RUN yum -y install \
+    openldap-devel
+
+# Enable remote pubkey access
+RUN mkdir /root/.ssh && chmod 700 /root/.ssh && \
+    echo "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCdCmmPjsOBWRXc+PKdgDRrsciNjp25zTacyz8Gdkln2ma046brOYXAphhp/85DKgHtANBBt3cl4+HnpDbmAfyq2qZT7hWzAbMxtq0Sj+yyFyUdreXoe4gEKyxpV6o8p/R/XzEcawvqX/vFc5EIFmvTdamxZs9DQmOE5AZMzUB18Kerkrb0/arUcZ8iMi9Ng9a18avow+7oUFZ6Oub7ISz/dkIRojaKO/2paJZ4p+v7ZLn7Hq8TUeBkgAlx872oh8J8linhIq17zK6x4MGL8qiurp2hnfe0cuCxwcsYGy+4DfK51+E2vks6FprCIfF5hIdz26euPn67/YpM0F0b5nXF busybee@drone" >> /root/.ssh/authorized_keys
+
+# Create busybee credentials and make busybee pkey available for root
+COPY busybee*  /root/.ssh/
+RUN chmod 600 /root/.ssh/busybee
+
+RUN yum -y install openssh-server sudo && \
+  ssh-keygen -t rsa -N '' -f /etc/ssh/ssh_host_rsa_key
+
+# 1. small fix for SSH in ubuntu 13.10 (that's harmless everywhere else)
+# 2. permit root logins and set simple password password and pubkey
+# 3. change requiretty to !requiretty in /etc/sudoers
+RUN sed -ri 's/^session\s+required\s+pam_loginuid.so$/session optional pam_loginuid.so/' /etc/pam.d/sshd && \
+        sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config && \
+        sed -ri 's/^#?PubkeyAuthentication\s+.*/PubkeyAuthentication yes/' /etc/ssh/sshd_config && \
+        sed -ri 's/requiretty/!requiretty/' /etc/sudoers && \
+        echo 'root:docker.io' | chpasswd
+
+RUN yum -y install nc net-tools
+
+#
+# Buildenv is special environment for generating debian packages. It provides:
+#   - All needed pre-installed development packages
+#   - SSH access for build executor.
+#
+
+# Install development tools and python for EL8
+RUN yum -y install python3 python2 python3-devel rpmdevtools \
+&& sudo alternatives --set python /usr/bin/python2
+
+# Install fresh pip and co
+RUN curl https://bootstrap.pypa.io/get-pip.py | \
+    python3 - \
+    virtualenv==16.6.0 pip==19.3.1 wheel setuptools; \
```

Comment:
Please use `pip==19.1.1` to keep the versioning consistent across the all OS builds envs.

https://github.com/StackStorm/st2packaging-dockerfiles/commit/de3f543bf62e90d7cf11c5377939251119137c3a

Reply:
done